webui/react/src/pages/JobQueue/RPStats.tsx

```diff
@@ -0,0 +1,27 @@
+import { Tooltip } from 'antd';
+import React from 'react';
+
+import OverviewStats from 'components/OverviewStats';
+import { RPStats } from 'types';
```

Comment:
**question + suggestion:** are there other types of **Pools** we have to consider in the future besides **Resource Pools**? If not, I think in the code base, it might be better to name them as `Pool` instead of `RP`.

Reply:
I don't think we have other types of poos. although we've had RP In the codebase before the pr. If we want to rename them to pool we should probably rename those as well.